app/controllers/course/admin/assessments/categories_controller.rb

```diff
@@ -0,0 +1,35 @@
+class Course::Admin::Assessments::CategoriesController < Course::Admin::Controller
+  load_and_authorize_resource :category,
+                              through: :course,
+                              through_association: :assessment_categories,
+                              class: Course::Assessment::Category.name
+  add_breadcrumb :index, :course_admin_assessments_path
+
+  def new
+  end
+
+  def create
+    if @category.save
+      redirect_to(course_admin_assessments_path(current_course),
+                  success: t('.success', title: @category.title))
+    else
+      render 'new'
+    end
+  end
+
+  def destroy
+    if @category.destroy
+      redirect_to(course_admin_assessments_path(current_course),
```

Comment:
Did a quick search and there are 28 uses of redirect_to() vs 57 for redirect_to. I guess we can standardise use of the DSL version?